src/components/Tabs/Tabs.module.scss

```diff
@@ -4,43 +4,41 @@
   width: 100%;
   display: flex;
   flex-direction: row;
-  justify-content: space-around;
   border-bottom: 1px solid $brand-lightGrey;
 }
 
 .tab {
-  flex-grow: 1;
-  text-align: center;
+  display: flex;
```

Comment:
so for the justify option - we would just set this back to flex-grow. Cool

Reply:
yessir